server/@types/dto/index.d.ts

```diff
@@ -3,6 +3,7 @@ declare module 'dto' {
     prisonNumber: string
     title: string
     steps: Array<AddStepDto>
+    targetCompletionDate?: string
```

Comment:
Re: my comment in your other PR I'm pretty sure this, and the same field in the form class, should be `Date` rather than string 🤔